lib/src/log_configuration.dart

```diff
@@ -2,11 +2,57 @@ part of mapbox_maps_flutter;
 
 /// A class that allows to configure Mapbox SDKs logging per application.
 final class LogConfiguration {
+    // create an instance here to initialize debug logging once
+  static LogConfiguration _instance = LogConfiguration._();
```

Comment:
if I get this right, we want this to be lazy init, should it be with `late` keyword here?
also imho it's much easier with a static boolean flag indicating if the log writer has been registered, instead of doing a lazy loading here